cmd/init.go

```diff
@@ -122,3 +139,24 @@ ks init app-name --api-spec=version:v1.7.1
 # specific build of Kubernetes to generate 'ksonnet-lib'.
 ks init app-name --api-spec=file:swagger.json`,
 }
+
+func genKsRoot(appName, appDir, ksDir string) (string, error) {
+	if appDir == "" {
+		return "", errors.New("invalid application directory")
```

Comment:
related to above, this might be `"invalid working directory"`. Consider renaming `appDir` and `ksDir` to `wd` and `appDir` respectively.